auto_round/autoround.py

```diff
@@ -239,8 +240,12 @@ def __init__(
             self.quant_block_list = find_matching_blocks(model, all_blocks, self.to_quant_block_names)
         self.cache_device = torch.device("cpu") if self.low_gpu_mem_usage else self.device
 
-        ##activation
-        self.act_group_size = act_group_size if act_group_size is not None else self.group_size
+        ##activation, default using per-tensor
+        self.act_group_size = act_group_size if act_group_size is not None else group_size
+        if self.act_group_size != 0:
+            logger.warning(
```

Comment:
refine this to your format and move to check_xxx function as we are going to support mxfp exporting